var/spack/repos/builtin/packages/roctracer-dev/package.py

```diff
@@ -36,6 +41,15 @@ def patch(self):
         filter_file('${CMAKE_PREFIX_PATH}/hsa',
                     '${HSA_RUNTIME_INC_PATH}', 'src/CMakeLists.txt',
                     string=True)
+        kwargs = {'ignore_absent': False, 'backup': False, 'string': False}
+        with working_dir('script'):
+            match = '^#!/usr/bin/python'
+            python = self.spec['python'].command
```

Comment:
```suggestion
            python = self.spec['python'].command.path
```
It might work as is, but this makes it more clear